.github/workflows/buildfrontend.yml

```diff
@@ -64,6 +64,14 @@ jobs:
       - name: Build project
         run: npx shadow-cljs release ${{ env.PROFILE }}
 
+      - name: Conditionally update app.js path in index (master)
+        if: endswith(github.ref, '/master')
+        run: sed -i 's/app.js/compiled\/app\/app.js/' 'resources/public/index.html'
+
+      - name: Conditionally update app.js path in index (develop)
+        if: endswith(github.ref, '/master')
```

Comment:
:)

(Other than that, looks good -- I forgot about the complication with master/develop.  I feel like there might be a better way around that, at the build level, but let's leave it for now.